pkg/workspace/inference/preset-inferences.go

```diff
@@ -103,6 +104,25 @@ func updateTorchParamsForDistributedInference(ctx context.Context, kubeClient cl
 	return nil
 }
 
+func updateParamsForDownload(wObj *v1beta1.Workspace, inferenceParams *model.PresetParam) {
```

Comment:
Do we have a chance to reuse this function https://github.com/kaito-project/kaito/blob/main/pkg/model/interface.go#L188

Reply:
Yes, let me update it